intl_translation_format/README.md

```diff
@@ -56,6 +56,169 @@ pub run intl_translation_format:generate
 ```
 
 ### Implement a new translation file format
-[TBD]
 
+The intl_translation_format brings the tools needed to support a new translation format. 
+You can extend the class TranslationFormat and complete the functions that will take care of the extracting of the messages
+and create templates. There are several classes to make this task easier: `MonoLingualFormat`, `BinaryMonolingualFormat`,
+`MultiLingualFormat`.
 
+As an example, this guide will saw you how the JSON file format is implemented.
+
+1. Learning about your format
+
+It is important to do some research about the format you are interested to implement. Some good questions would be:
+ 1. Is it a standard? 
```

Comment:
Maybe -  Is there a standard or other definitive document that can tell us how to resolve any questions about the implementation?

Or is there another reason they care if it's a standard?